packages/framework/build.xml

```diff
@@ -106,6 +106,14 @@
             <arg value="assets:install"/>
             <arg value="${path.root}/web/"/>
         </exec>
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="elfinder:install"/>
+        </exec>
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:elfinder:post-install"/>
+        </exec>
```

Comment:
Is it worth to mention this change in upgrade notes?

Reply:
Yes, there are changed files in project-base

Reply:
This file is not from a project-base :)